operator/e2e/e2e_test.go

```diff
@@ -75,7 +77,6 @@ func TestE2E(t *testing.T) {
 		{
 			name: "random-yaml-nonroot",
 			f: func(t *testing.T) {
-				t.Parallel()
```

Comment:
I will remove these from this PR, and open a separate PR together with some other e2e changes to support k3d.

Reply:
We could also use something like gotestfmt in this case to make parallel outputs nicer.